### PR TITLE
Enums, ID and targeting fixes and etc

### PR DESCRIPTION
**Pull request by ReddeR1337**
Enums, item ID, targeting and other fixes

### CODE REVIEW

**Comment by ReddeR1337** [comment]
Pull №8 = version 438

**Comment by ReddeR1337** [comment]
Fix CurrentTarget

File: Components/Combat/DefaultTargetingProvider.cs [hunk 1]
```diff
@@ -244,38 +243,37 @@ public bool IsInRange(Vector3 position, TrinityPower power)
...
-            if (Core.Player.IsInBossEncounter && currentTarget != null && currentTarget.ActorSnoId == (int) SNOActor.Belial)
+            // Handle Belial differently, he's never in LineOfSight. Replace CurrentTarget to CurrentTarget for fix error
+            if (Core.Player.IsInBossEncounter && CurrentTarget != null && CurrentTarget.ActorSnoId == (int) SNOActor.Belial)
                 return distance <= rangeRequired;
 
             return distance <= rangeRequired && IsInLineOfSight(position);
         }
 
-        public bool IsInLineOfSight(TrinityActor currentTarget)
+        public bool IsInLineOfSight(TrinityActor CurrentTarget)
```
Comment (jakobbotsch) [inline Components/Combat/DefaultTargetingProvider.cs]: Please revert this change (MS naming convention uses camelCase for parameter names).

**Comment by jakobbotsch** [comment]
There are lots of changes to seemingly unrelated files in this PR (README.md, new profiles like DontDeleteFiles.xml, NEC_RathmaCDRSimulacrum4.cs, NEC_RathmaCDRSimulacrum4Settings_en.xaml).
Can you please clean this up?

**Comment by ReddeR1337** [comment]
Ok, i close this pull, del other shit and start new
